contrib/apelscripts/condor_batch.sh

```diff
@@ -8,21 +8,12 @@ output=batch-$(date -u --date='yesterday' +%Y%m%d )-$(hostname -s)
 # Build the filter for the history command
 CONSTR="EnteredCurrentStatus >= $yesterday && EnteredCurrentStatus < $today && RemoteWallclockTime !=0"
 
-function die() { echo "$0 aborted when reading config"; exit 1; }
-trap die ERR;
-if [ $# -gt 0 ]; then
-  source $1
-fi
-trap '' ERR
-
-[[ -z "$SCALING_ATTR" ]] && SCALING_ATTR=MachineAttrRalScaling0
-[[ -z "$BATCH_HOST" ]] && BATCH_HOST=`hostname`
-[[ -z "$OUTPUT_LOCATION" ]] && OUTPUT_LOCATION=/var/lib/condor-ce/apel/
-
-OUTPUT_FILE=$OUTPUT_LOCATION/$output
+HISTORY_SUFFIX='-format "\n" EMPTY'
+SCALING_ATTR=$(condor_ce_config_val APEL_SCALING_ATTR)
+[ $? -eq 0] && HISTORY_SUFFIX="-format "%v|" ${SCALING_ATTR} ${HISTORY_SUFFIX}"
 
 TZ=GMT condor_history -constraint "$CONSTR" \
-    -format "%s_${BATCH_HOST}|" ClusterId \
+    -format "%s_$(condor_ce_config_val APEL_BATCH_HOST)|" ClusterId \
```

Comment:
What happens if this or APEL_OUTPUT_DIR do not exist or have bogus values?

Reply:
We provide default config values in `/usr/share/condor-ce/config.d/` so I'm not as worried about the values not existing

Reply:
What is the behavior when the condor ce config is unparseable, for example due to a syntax error?